README.md

```diff
@@ -49,6 +49,21 @@ Launch the OpenFin runtime:
 npm run openfin
 ```
 
+### NPM Tasks
+
+Run webpack-dev-server with the development config: ``` npm run start ```
+
+Launch OpenFin with the development setup: ``` npm run openfin:dev ```
+
+Launch OpenFin with the production setup: ``` npm run openfin:prod ```
+
+Build and serve the production bundle: ``` npm run prod ```
+
+Lint and run the tests: ``` npm run ci ```
+
+Run the tests and watch the test directory for changes: ``` npm run test:watch ```
```

Comment:
Could also group the test commands with a subheading.

Worth listing `npm test` too?